gatsby/content/blog/2021/06/2021-06-11-twim.mdx

```diff
@@ -160,9 +160,9 @@ Synapse is a popular homeserver written in Python.
 >
 > Docker-based development environment for Matrix. Provides a local sandbox with the following pre-configured services:
 >
-> synapse: the reference homeserver implementation
-> synapse-admin: homeserver admin UI
-> element: a web-based Matrix client
+> * synapse: the reference homeserver implementation
+> * synapse-admin: homeserver admin UI
+> * element: a web-based Matrix client
```

Comment:
To sanity check this, the permalink has the list as well, https://matrix.to/#/!jxlRxnrZCsjpjDubDX:matrix.org/$OM-Kxwdqz_Jiex9eMJxTBhzCVwVeZAXYE_HIm3w3AZs?via=matrix.org&via=privacytools.io&via=mozilla.org

![](https://user-images.githubusercontent.com/558581/121735543-e5524000-cabb-11eb-8ed8-bdf9237858d6.png)